README.md

```diff
@@ -29,26 +29,103 @@ git clone https://github.com/sap/service-fabrik-boshrelease
 cd service-fabrik-boshrelease
 ```
 
+### Deploying Service-Fabrik
 
-### Deploy from Sources
+#### Creating and uploading release:
 
-If you need to modify the sources (beyond the example manifest):
+  ##### Deploying from Sources
+
+  If you need to modify the sources (beyond the example manifest):
+  ```shell
+  ./scripts/update
+  bosh create release --force && bosh upload release # repeat these commands on every change
+  ```
+
+  #### Deploying from Release
+
+  If you do not need to modify the sources (beyond the example manifest):
+  ```shell
+  bosh upload release $(ls -1rv releases/service-fabrik/service-fabrik-*.yml | head -1)
+  ```
+
+#### Uploading the cloud-config (for AWS):
+Then, we need to upload the cloud-config required for service-fabrik on bosh.
+For that, we need to update the vars-files for the cloud-config. 
+This can be done in 2 ways, providing the parameters through command line or editing the vars-file directly.
+
+The vars file to be edited is `cloud-config-aws-vars.yml`. It can be found in the `templates` directory.
+Once the vars file is filled with proper details, the cloud-config can be uploaded:
 ```shell
-./scripts/update
-bosh create release --force && bosh upload release # repeat these commands on every change
 cd templates
-./make_manifest warden
-bosh -n deploy
+bosh –e bosh upload-cloud-config --vars-store=cloud-config-aws-vars.yml cloud-config-aws.yml
 ```
 
-### Deploy from Release
+Alternatively, the parameters can be passed through the command line as below:
+```shell
+cd templates
+bosh –e bosh upload-cloud-config -v availability_zone1="value1" \
+-v availability_zone2="value2" \
+-v availability_zone3="value3" \
+-v network_id1="value4" \
+-v network_id2="value5" \
+-v network_id3="value6" \
+-v network_id4="value7" \
+-v network_id5="value8" \
+-v network_id6="value9" \
+-v security_group1="value10" \
+-v security_group2="value11" \
+-v security_group3="value12" \
+-v security_group4="value13" \
+-v security_group5="value14" \
+-v security_group6="value15" \
+cloud-config-aws.yml
+```
 
-If you do not need to modify the sources (beyond the example manifest):
+#### Uploading the cloud-config (for boshlite):
+For bosh-lite, you can upload cloud-config in the following manner:
 ```shell
-bosh upload release $(ls -1rv releases/service-fabrik/service-fabrik-*.yml | head -1)
 cd templates
-./make_manifest warden
-bosh -n deploy
+bosh –e bosh upload-cloud-config cloud-config-boshlite.yml
+```
+
+#### Deployment:
+Once the release and cloud-config has been uploaded to bosh, we can deploy service-fabrik.
+
+First we need to generate the services catalog file. Run the render script present in the templates directory.
+```shell
+./render
```

Comment:
I'm curious...wouldn't the dashboard link need to be adjusted for boshlite or any other cf?

Reply:
As discussed with you, this seems to be a bit complicated task for now. Have raised a separate issue for this and will take it up as a separate PR. https://github.com/cloudfoundry-incubator/service-fabrik-boshrelease/issues/43

Reply:
Please take it up after this.